Source/HyprMXAdapter.swift

```diff
@@ -116,6 +123,12 @@ final class HyprMXAdapter: PartnerAdapter {
     /// Indicates if the user is subject to COPPA or not.
     /// - parameter isChildDirected: `true` if the user is subject to COPPA, `false` otherwise.
     func setCOPPA(isChildDirected: Bool) {
+        // We map the COPPA setting to HyprMX's ageRestrictedUser setting based on their description
+        // of its intended use, for instance "If the user requires child-directed treatment under
+        // applicable laws and policies, set this parameter to true."
+        // More info at https://documentation.hyprmx.com/ios-hyprmx-sdk/#initialization-api
+        UserDefaults.standard.set(isChildDirected, forKey: AGE_RESTRICTED_USER_KEY)
```

Comment:
Do we need to save this in `UserDefaults`? Does the SDK send this to adapters at every startup? I'm wondering if we can just save this in a var instead of `UserDefaults`.

Reply:
We received [feedback from HyprMX](https://chartboost.slack.com/archives/C051EJBP6ES/p1688752269185679) that that the `ageRestrictedUser` flag should be set at init, and our 4.x SDK doesn't pass privacy settings to the adapters until after calling `.setUp()`, so we need to cache the last known value and apply it in `HyprMX.initialize()`

Reply:
Ah gotcha, that makes sense. Can we add that as a comment here? I see we have this comment above:
> // consentStatus will be updated by setGDPR & setCCPA after init

But I'm wondering if we can expand on that that make it a little more clear.